Source/Core/VideoCommon/FramebufferShaderGen.cpp

```diff
@@ -225,50 +238,52 @@ std::string GenerateScreenQuadVertexShader()
 
 std::string GeneratePassthroughGeometryShader(u32 num_tex, u32 num_colors)
 {
-  std::stringstream ss;
+  std::ostringstream ss;
   if (GetAPIType() == APIType::D3D)
   {
-    ss << "struct VS_OUTPUT\n";
-    ss << "{\n";
+    ss << "struct VS_OUTPUT\n"
+          "{\n";
     for (u32 i = 0; i < num_tex; i++)
       ss << "  float3 tex" << i << " : TEXCOORD" << i << ";\n";
     for (u32 i = 0; i < num_colors; i++)
       ss << "  float4 color" << i << " : COLOR" << i << ";\n";
-    ss << "  float4 position : SV_Position;\n";
-    ss << "};\n";
-    ss << "struct GS_OUTPUT\n";
-    ss << "{";
+    ss << "  float4 position : SV_Position;\n"
+          "};\n";
+
+    ss << "struct GS_OUTPUT\n"
```

Comment:
Why is this using `ss << ` instead of making it one long string like you do above?

Reply:
Because the string above it is related to a completely different struct declaration. For visual clarity I've kept them apart from one another.

Reply:
Sure, but keeping a blank line between those would still keep the separation clear, still without the need to have a separate `<<`. IMO

Reply:
There is no difference here. The use of `<<` itself isn't an issue, especially given it's going to be moved over to fmt in a follow-up change, and I'd still prefer to keep the struct declarations separate from one another in that context as well.